TestViewer/MainWindow.xaml.cs

```diff
@@ -203,7 +203,7 @@ private void DrawEcg (double cur_time)
             try {
                 ecg = m_source.GetECG();
 
-                if (ecg.samples.Length == 0) {
+                if (ecg.samples == null) {
```

Comment:
Just to be 100% sure: Can you please confirm that `ecg.samples` will indeed be `null` in C# if the returned EcgSeries object is empty?

Also, would it be possible for you to also check that the logic also works as expected for EcgSeries objects created in C#? I think you can do that with something like `EcgSeries ecg = default(EcgSeries);`

Reply:
With C# default EcgSeries, the arrays are null.
![image](https://user-images.githubusercontent.com/16581681/165571551-b3a66d09-a6db-480c-ad17-41f3c3f93776.png)

Reply:
With default C++ EcgSeries, the arrays are also null when called from C#

![image](https://user-images.githubusercontent.com/16581681/165573764-9a33c3fc-f495-49bf-8130-8db2a202dd7c.png)